src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf),
+    url(/fonts/Avenir-Heavy.ttf),
+    url(/fonts/Avenir.ttc);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
```

Comment:
don't set these properties via * selector.
use * only for setting box-sizing property